src/Dashboard/Widget.php

```diff
@@ -53,6 +55,42 @@ class Widget
 {
     public static $animation_duration = 1000; // in millseconds
 
+    public static $palette_10 = [
```

Comment:
Could this be used by another class ? If not, we could change it to private.

Reply:
this point is addressed in https://github.com/glpi-project/glpi/pull/12884